src/styles/card/image.scss

```diff
@@ -0,0 +1,9 @@
+.images {
+  background-image: url(../images/imac.jpeg);
```

Comment:
The background image should be set in the HTML file, not in the CSS. This is because the image is not a part of the styling, it's a content of your website. So it should be placed in the HTML file.